### PR TITLE
Feat/add ruff less rules

### PR DESCRIPTION
**Pull request by lmontier-pass**
# DE PR

## Describe your changes

Add ruff as a linter in 
* the project
* the CI
* pre-commit

## Jira ticket number and/or notion link

https://www.notion.so/passcultureapp/880351701699470cb129e4e0ad6bbcba?v=cfbe4b16778b47218f252bd1b5add4a0&p=6ecacaf3def94776a50738e68005e212&pm=s

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] My code passes CI/CD tests
- [x] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,3 +65,14 @@ create_microservice:
 	git add .
 	git commit -am "Add $(MS_NAME) microservice"
 	make clean_install
+
+ruff_fix:
```
Comment (cdelabre) [inline Makefile:69]: Peut on faire une version simple avec ruff only pour lemoment ?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## ruff
+# Recommended ruff config for now, to be updated as we go along.
+[tool.ruff]
+target-version = 'py310'
```
Comment (cdelabre) [inline pyproject.toml:4]: 3.8

**Review by cdelabre — COMMENTED** [review]
Suggestions

**Review by cdelabre — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (lmontier-pass) [reply Makefile:69]: vu ensemble, je fais ça dans un prochain ticket!

**Pull request merged**